src/static/css/pages/careers.less

```diff
@@ -0,0 +1,3 @@
+.students-and-graduates .media_image {
+    width: 150px
```

Comment:
This really should be consistent against all the Career pages. We're creating so much code for edge-cases that's going to be a pain in the neck to test.